segregation/non_spatial_indexes.py

```diff
@@ -432,7 +444,11 @@ def _isolation(data, group_pop_var, total_pop_var):
     X = x.sum()
     xPx = np.nansum((x / X) * (x / t))
     
-    core_data = data[['group_pop_var', 'total_pop_var']]
+    if (str(type(data)) != '<class \'geopandas.geodataframe.GeoDataFrame\'>'):
```

Comment:
this seems like a kinda convoluted way to do the type check, but not a huge deal

Reply:
Yes, I agree! Is there a shorter way to do that? I tried to use isinstance, but it assumes the way the user import geopandas:

![image](https://user-images.githubusercontent.com/22593188/57261460-c2234f80-701c-11e9-811a-049e0a05879b.png)

Reply:
However, the current way works for any different way to import geopandas.
BTW, I really like the `asserts` you use in geosnap for conditions... I think I'll refactor to that kind later.